eng/testing/BionicRunOnDevice.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+currentDirectory=$(realpath `dirname $0`)
+currentTest=$(basename $0 .sh)
+runtimeExeArg=${1:-}
+runtimeExe=${2:-}
+extraLibsArg=${3:-}
+extraLibsDir=${4:-}
+if [ $# -eq 0 ]; then
+	echo "Usage: ${currentTest}.sh -r /path/to/dotnet [-l /path/to/preloadlibs]"
+	exit 0
+fi
```

Comment:
It'd be nice if the argument parsing followed the style in https://github.com/dotnet/runtime/blob/main/eng/testing/RunnerTemplate.sh. Or does the device shell not support while/case/tr etc.?

Reply:
there are also a lot of Shellcheck suggestions for this file.

Reply:
It's a pretty basic `sh` implementation, I assume BusyBox. There's no Bash at least.

I'd have to test a more sophisticated shell script to see what works.